### DISCUSSION TITLE
能否显示视频所属分区？

### DISCUSSION THREAD

**Discussion by SherkeyXD** [the1812/Bilibili-Evolved#3487] [提问 / Q&A] [answered]
有没有这个功能的脚本捏

**Comment by the1812** [comment]
下面的 tag 就是分区

**Comment by Geeyun-JY3** [comment, marked as answer]
tag 行里从左往右再从上往下看，除去带的话题和频道 tag，接下来的两个 tag 就是分区，有个高亮出分区的样式：https://greasyfork.org/scripts/431798

另外如果你想要在视频标题下方像以前那样显示分区，可以用上面这个样式作者的一个脚本：https://greasyfork.org/scripts/398655
